src/OpenSearch/Client.php

```diff
@@ -1706,7 +1706,7 @@ public function searchTemplate(array $params = [])
      * $params['offsets']          = (boolean) If `true`, the response includes term offsets. (Default = true)
      * $params['payloads']         = (boolean) If `true`, the response includes term payloads. (Default = true)
      * $params['positions']        = (boolean) If `true`, the response includes term positions. (Default = true)
-     * $params['preference']       = (string) Specifies the node or shard on which the operation should be performed. See [preference query parameter]({{site.url}}{{site.baseurl}}/api-reference/search-apis/search/#the-preference-query-parameter) for a list of available options. By default the requests are routed randomly to available shard copies (primary or replica), with no guarantee of consistency across repeated queries.
```

Comment:
Removing a space doesn't look right.

Reply:
https://github.com/opensearch-project/opensearch-api-specification/blob/main/spec/namespaces/_core.yaml#L5108-L5110

Reply:
I created this PR to resolve the issue #382